pkg/build/builder/daemonless.go

```diff
@@ -130,18 +130,13 @@ func buildDaemonlessImage(sc types.SystemContext, store storage.Store, isolation
 		}
 	}
 
-	var transientMounts []imagebuildah.Mount
+	var transientMounts []string
 	if st, err := os.Stat("/run/secrets"); err == nil && st.IsDir() {
 		// Add a non-recursive bind of /run/secrets, to pass along
 		// anything that the runtime mounted from the node into our
 		// /run/secrets, without passing along API secrets that were
 		// mounted under it exclusively for the builder's use.
-		transientMounts = append(transientMounts, imagebuildah.Mount{
-			Source:      "/run/secrets",
-			Destination: "/run/secrets",
-			Type:        "bind",
-			Options:     []string{"ro", "nodev", "noexec", "nosuid"},
-		})
+		transientMounts = append(transientMounts, "/run/secrets:/run/secrets:ro,nodev,noexec,nosuid")
```

Comment:
Reviewer note: this specification eventually gets turned into a recursive bind mount by the newer API.  Is that a breaking change, or should I just update the comment to reflect the new behavior?

Reply:
On the surface this doesn't appear to be a breaking change. The build controller doesn't add any content here, and from what I can gather recursive bind mounting is a desired outcome. No?

Reply:
If the build controller doesn't depend on that, then it's fine.  I'll remove the bit from the comment about it being non-recursive.